docs/basics/1d_hist.rst

```diff
@@ -238,3 +238,22 @@ To easily get the values and the uncertainties of the comparison, the :func:`get
     values, lower_uncertainties, upper_uncertainties = get_comparison(
         h1, h2, comparison="ratio"
     )
+
+.. _1d-profile-plot-label:
+Mean histogram (profile plot)
+=============================
+
+The function :func:`make_hist() <plothist.histogramming.make_hist>` returns a counting histogram, where each bin holds the total number of data points or their weighted sum.
+The returned histogram is an object defined in the ``boost_histogram`` `package <https://boost-histogram.readthedocs.io/>`_.
```

Comment:
Remove or rephrase "The returned histogram is an object defined in the boost_histogram"